flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -709,22 +730,9 @@ private ApplicationReport startAppMaster(
 			systemShipFiles.add(file.getAbsoluteFile());
 		}
 
-		//check if there is a logback or log4j file
-		File logbackFile = new File(configurationDirectory + File.separator + CONFIG_FILE_LOGBACK_NAME);
-		final boolean hasLogback = logbackFile.exists();
-		if (hasLogback) {
-			systemShipFiles.add(logbackFile);
-		}
-
-		File log4jFile = new File(configurationDirectory + File.separator + CONFIG_FILE_LOG4J_NAME);
-		final boolean hasLog4j = log4jFile.exists();
-		if (hasLog4j) {
-			systemShipFiles.add(log4jFile);
-			if (hasLogback) {
-				// this means there is already a logback configuration file --> fail
-				LOG.warn("The configuration directory ('" + configurationDirectory + "') contains both LOG4J and " +
-						"Logback configuration files. Please delete or rename one of them.");
-			}
+		final String logConfigFilePath = configuration.getString(YarnConfigOptions.APPLICATION_LOG_CONFIG_FILE);
```

Comment:
I just tried submitting on YARN with HA (zookeeper). It doesn't work properly because the app master is using a different zk namespace than the client.

Master:
```
2019-10-29 16:12:46,957 INFO  org.apache.flink.runtime.util.ZooKeeperUtils                  - Using '/flink/default' as Zookeeper namespace.
```

client is using `/flink/<yarn_app_id>`

Reply:
Related commit 2eba494890f7aa5056ce6ddf8ddcd804e4ca07e3

Reply:
@GJL do you run in session mode or per-job mode?

Reply:
I tried session mode.

Reply:
I think I fixed the issue. The problem seems to be that the default zk namespace value was getting in the way when we were checking `if (namespace == null)...` in the `YarnClusterDescriptor.startAppMaster`.

Reply:
Smoke tests are passing.

Reply:
Thanks a lot @GJL for testing this. I will merge.